src/formElementPlugins/ApplicationViewWrapperNEW.tsx

```diff
@@ -216,12 +216,12 @@ const calculateValidationState = async ({
   const validationResult = validationExpression
     ? await validate(validationExpression, validationMessage as string, evaluationParameters)
     : { isValid: true }
+  console.log(responses, isRequired && isStrictPage && !responses?.thisResponse, {
+    validationMessage: validationMessage || strings.VALIDATION_REQUIRED_ERROR,
+  })
   if (!validationResult.isValid) return validationResult
-  if (
-    isRequired &&
-    isStrictPage &&
-    (responses.thisResponse === undefined || responses.thisResponse === null)
-  )
+  // !responses.thisResponse, check for null, undefined, empty string
+  if (isRequired && isStrictPage && !responses?.thisResponse)
```

Comment:
Maybe this was a safe-guard for elements that hold `thisResponse` == `false` ?

Reply:
hmm it's always been a text field (i changed it to this because empty string was failing to pass this test, and also checking for empty field makes the if statement extra large).